platform-sdk/swirlds-logging/src/testFixtures/java/com/swirlds/logging/test/fixtures/internal/LoggerMirrorExtension.java

```diff
@@ -32,9 +33,11 @@ public void interceptTestMethod(
             ExtensionContext extensionContext)
             throws Throwable {
         try (final LoggingMirrorImpl loggingMirror = new LoggingMirrorImpl()) {
+            DefaultLoggingSystem.getInstance().addHandler(loggingMirror);
```

Comment:
we should do a try-finally for adding/removing the handler

Reply:
Ended, try-finally inside a try seemed ugly, but it leaves a lot of scenarios without removing the handler